docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,72 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_login": "foo",
+            "key_path": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        }
+    }
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
+
+``ssh_login``
+  ssh login to remote machine
+
+``key_path``
+  A path to the private key that will be used as identity file for ssh. When ``auth_type`` is
+  ``publickey`` this is a required config. The key has to be readable by user ``apache``.
+
+``ssh_password``
+  Password is required when ``auth_type`` is 'password'.
+
+``host``
+  remote hostname
+
+``root``
+  Absolute path to the remote root directory where all the data (content and published content)
+  lives. Remote equivalent to ``/var/lib/pulp``.
+
+Optional configuration
+----------------------
+
+``skip_fast_forward``
+  If true, rsync distributor will publish all content of repository.
+
+``content_units_only``
+  If true, the distributor will publish content units only (e.g. /var/lib/pulp/content). The
+  symlinks of a published repository will not be rsynced.
+
+``skip_repodata``
+  If true, repodata will be omitted from the publish.
+
+``delete``
+  If true, appends --delete to the rsync command for symlinks and repodata so any old files no longer present in
+  the local published directory are removed from remote server.
+
+``force_full``
```

Comment:
How is this different from `skip_fast_forward`?

I think the distinction is that `force_full` will cause the publish to occur even if there are no new units to publish since the last published occurred. That is different than `skip_fast_forward` which causes all units to be published when there are any units to publish.

Is ^ right?